packages/cli/src/concurrency/concurrency-queue.ts

```diff
@@ -1,9 +1,14 @@
 import { Service } from 'typedi';
-import { EventEmitter } from 'node:events';
-import debounce from 'lodash/debounce';
+import { TypedEmitter } from '@/TypedEmitter';
+
+interface Events {
```

Comment:
Interfaces are subject to declaration merging so if we ever export we can accidentally create a big `Events` interface. Let's use `type` instead? Same elsewhere.

Reply:
Also for clarity and easier searching I'd rename based on the service, e.g. `ConcurrencyEvents`